packages/eas-cli/src/commandUtils/EasCommand.ts

```diff
@@ -127,14 +132,33 @@ export default abstract class EasCommand extends Command {
    */
   private readonly sessionManager = new SessionManager();
 
+  override get ctor(): typeof EasCommand {
+    return this.constructor as typeof EasCommand;
+  }
```

Comment:
What's this? :o

Reply:
I'm guessing this is needed to get access to static members of the derived class.
I think it could use some comment on how it works and maybe instead of exposing constructor it would be better to return just  contextDefinition e.g.
```
  override get resolveCommandContextDefinition(): ContextInput {
    return this.constructor.contextDefinition
  }
``